pydss/flopper.py

```diff
@@ -106,7 +106,7 @@ def tick(self, bid_id,now):
     self.bids[bid_id].end = now+ self.tau;
 
 
-def dent(self, bid_id, lot, bid):
+def dent(self, bid_id, lot, bid, sender):
     require( (self.live == 1), "Flopper/not-live")
     require( (self.bids[bid_id].guy != 0),"Flopper/guy-not-set")
     require ( (self.bids[bid_id].tic > self.now or self.bids[id].tic == 0),"Flopper/already-finished-tic")
```

Comment:
here as well, using `self.now`
also you've got an extra set of parentheses around the condition